crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -18,8 +19,17 @@ pub struct CybersourcePaymentsRequest {
 }
 
 #[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
 pub struct ProcessingInformation {
     capture: bool,
+    capture_options: Option<CaptureOptions>,
+}
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct CaptureOptions {
+    capture_sequence_number: i32,
+    total_capture_count: i32,
```

Comment:
Do you need these fields to be signed integers, or can they be unsigned integers instead?